src/nucname.h

```diff
@@ -96,6 +99,7 @@ namespace nucname
     {
        nucwas = wasptr;
        nucnow = nowptr;
+       setNaNEstr();
```

Comment:
Do we need this function, or can we just assign the value of this new member variable in `what()`?

Reply:
The `const` after `what()` prevents setting the value of member variables like `NaNEstr`, so the only way is to do it in the constructors.